new-site/src/components/examples/TextExample.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TextExample = ({ color = 'var(--color-black-90)', size = 'var(--fontsize-heading-m)', style = {}, weight = 400 }) => (
+  <span
+    style={{
```

Comment:
Could we simplify this little by changing:

size -> fontSize
weight -> fontWeight